config/rbac/role.yaml

```diff
@@ -386,6 +386,19 @@ rules:
   - get
   - patch
   - update
+- apiGroups:
```

Comment:
Once we land this the SRIOV operator is required right? Perhaps update update README with this requirement?